.coveragerc

```diff
@@ -1,4 +1,5 @@
-[report]
-omit =
-    */.tox/*
-    */tests/*
+[run]
+branch = true
+source =
+    num2words
+    tests
```

Comment:
@btharper Do we need to report the coverage on the tests?

Reply:
Running coverage against the tests was how I found the extra code in [tests/test_fi.py](https://github.com/savoirfairelinux/num2words/pull/220/commits/62de819126ae3c452c01c27ef599fe6ed2e70bbf#diff-7372b309c37da132dc6962d7f0970ed6), so I definitely think there's value in the information. However measuring test coverage in a continuous environment (especially with 100% requirement enforced in tox.ini in the same PR) is probably overkill, but I wasn't sure how to manage it in an automated fashion. Especially if any tests end up with any  code specific to the travis job it runs in (python2/3, pypy, or otherwise), 100% coverage of test code may not be certain. It may be better suited to a manual test only run as needed.

Reply:
Ok.